### PR TITLE
48 add forgot password feature

### PR DESCRIPTION
**Pull request by brokelyn**

### CODE REVIEW

File: src/lib/components/ChatMessage.svelte [hunk 1]
```diff
@@ -29,7 +29,7 @@
   </div>
 </div>
 <div class="p-4 md:p-8 bg-accent bg-opacity-10 flex justify-center">
-  <div class="max-w-screen-md w-full flex gap-4 md:gap-8">
+  <div class="max-w-screen-lg w-full flex gap-4 md:gap-8">
```
Comment (brokelyn) [inline src/lib/components/ChatMessage.svelte:32]: off topic change but I like it way more when the chat window is a bit wider

**Comment by brokelyn** [comment]
I will wait until my Postmark account got verified and update some of the email stuff. But the rest should be fine for review

File: src/routes/app/settings/teams/new/+page.svelte [hunk 2]
```diff
@@ -52,7 +52,7 @@
       <button
         type="submit"
         class="rounded-md bg-indigo-500 px-3 py-2 text-sm font-semibold text-white shadow-sm hover:bg-indigo-400 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-500"
-        >Crreate</button
+        >Create</button
```
Comment (tonioriol) [inline src/routes/app/settings/teams/new/+page.svelte:55]: ahhh nice, that was annoying

File: src/routes/forgot-password/+page.server.ts [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+          error: 'There is no user with that email address',
+        })
+      }
+
+      const uuid = randomUUID()
+      user = await updateResetTokenToUser(user.id, uuid)
+      const resetUrl = `${url.origin}/reset-password/${uuid}`
+
+      await postmark.sendEmail({
+        From: 'philipp.krause@prototyp.se',
```
Comment (tonioriol) [inline src/routes/forgot-password/+page.server.ts:32]: Is that on purpose, or you forgot?

File: src/routes/forgot-password/+page.svelte [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+      class="mt-6 text-center text-2xl font-bold leading-9 tracking-tight text-gray-900 dark:text-gray-100"
+    >
+      Reset your password
+    </h2>
+  </div>
+
+  <div class="mt-10 sm:mx-auto sm:w-full sm:max-w-[480px]">
+    <div class="bg-white dark:bg-gray-900 px-6 py-12 shadow sm:px-12">
+      <Alert
+        type={(form?.error && 'error') || (form?.success && 'success')}
```
Comment (tonioriol) [inline src/routes/forgot-password/+page.svelte:32]: nice

**Review by tonioriol — APPROVED** [review]
Looking really nice :)

**Pull request merged**